changelog.md

```diff
@@ -1,4 +1,4 @@
-### 1.1.1 - Unreleased
```

Comment:
Nit: I think this should tecnically be 1.2.0 since we added an API.

Reply:
Fair point, fixed in https://github.com/amqp/rhea-promise/pull/81/commits/9310496007d8e2fa518bdc93ccdb979a7c932f8c